src/Sentry.Unity/SentryInitialization.cs

```diff
@@ -22,7 +22,8 @@ public static class SentryInitialization
         // Temp event capture infra
         public static IEventCapture EventCapture = new EventCapture();
 
-        private static readonly IUnityLogMessageFilter _typeConditionFilter = new TypeConditionFilter();
+        private static readonly ErrorTimeDebounce _errorTimeDebounce = new();

```

Comment:
We'll need to refactor these `static` fields later.
Whatever we're doing not on the method annotated with `RuntimeInitializeOnLoadMethod` will need to become callable by user-code, optionally. If they decided to opt-out from this auto-init.

The "Integration" that will hook into the logger callback will make usage of these instances, so they might be passed as a dependency to such integration. Today everything is just hacked on this file.

Reply:
100%! I already started looking into https://github.com/getsentry/sentry-unity/issues/66, my hope it will be covered there)